Isabel-Gomez/challenge/tour-of-heroes-react/src/components/common/Card.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react';
+import Details from '../Details';
+
+const Card = ({ data }) => {
+  const [showDetails, setShowDetails] = useState(false);
+
+  return (
+    <>
+      <li>
+        <button type="button" onClick={() => setShowDetails(!showDetails)}>+</button>
+        {data.name}
+      </li>
+      {showDetails && <Details data={data} />}
```

Comment:
I think you can put the ` {showDetails && <Details data={data} />}` inside the li